### ISSUE TITLE
tailblazor.dev is not loading

### ISSUE THREAD

**Issue by umanivannan-conga** [McNerdius/TailBlazor-Templates#26]
I'm not sure whether this is the right place to create this, but creating this anyway.

![image](https://user-images.githubusercontent.com/63358749/167436712-b25b18aa-03a4-48c9-bb2b-1163a9834e53.png)

**Comment by McNerdius** [comment]
Not seeing any issues on azure portal, but this could be the issue:  

https://stackoverflow.com/questions/66221439/blazor-webassembly-application-fails-to-load-due-to-integrity-errors

(Anonymous Browsing/Antivirus/Firewall blocking `dll` files)

The [documented workaround](https://docs.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/webassembly?view=aspnetcore-6.0#change-the-filename-extension-of-dll-files) is a bit hacky, though.  Do any of the above scenarios apply ?

**Comment by umanivannan-conga** [comment]
Yeah i think its probably some security settings in my company provided laptop, I'm able to access the site from my mobile without issues.

**Comment by McNerdius** [comment]
OK thanks for the info.  I'll be looking around for a more elegant solution than the published workaround this week, hopefully update tailblazor.dev & tailblazor.net repos this weekend, and mention it somewhere on tailblazor.dev.

**Comment by McNerdius** [comment]
So, i've applied the MS suggested workaround to the `tailblazor.dev` repo ([issue](https://devblogs.microsoft.com/dotnet/custom-deployment-layout-for-blazor-webassembly-apps/) / [PR](https://github.com/McNerdius/TailBlazor/pull/130/files)).  The (hopeful) fix is hosted [here](https://polite-sky-006af1d1e-130.westus2.azurestaticapps.net/), pending confirmation that the fix is a real, actual fix & PR merge.  Then the fix gets applied to this repo, and so on.

**Comment by umanivannan-conga** [comment]
hI @McNerdius, Checked the fix, It looks like .bin extensions also seems to be blocked by our security vendor

![image](https://user-images.githubusercontent.com/63358749/168252777-7992a202-1aff-42f3-95cf-bffa6941d34b.png)

**Comment by McNerdius** [comment]
Bummer :(

I have merged the "rename dll to bin" non-fix into `tailblazor.dev` - hopefully it will be the basis for a *real* fix, which will make its way here, into the `tailblazor-templates`.  The hard part here is not knowing how to reproduce this security behavior on my end.

I'm working on a time-constrained commissioned project at the moment so i can't give this my full attention.  I may be busy with that project as far out as mid-July(!), but i will be googling/stack-overflowing/discord-inquiring about it when i can. 

In the meantime, i'm pinning this and [#129](https://github.com/McNerdius/TailBlazor/issues/129) on the other repo.  Hopefully there is a more robust yet still simple solution to be found.  I wouldn't want to add a bunch of code to `tailblazor-templates` that only applies to one particular scenario, but it's certainly something i'd include in `tailblazor.dev` and its notes.
